test/helpers/cons.go

```diff
@@ -165,7 +165,7 @@ const (
 	// connecting or transferring data. CurlMaxTimeout should be at least 5
 	// seconds longer than CurlConnectTimeout to provide some time to
 	// actually transfer data.
-	CurlMaxTimeout = 8
+	CurlMaxTimeout = 20
```

Comment:
The timeout is used by `CurlFail()`, which is sometimes _supposed_ to fail. Won't that increase the waiting time for the tests when we expect the connexion to fail?

Reply:
@qmonnet this should be handled by `CurlConnectTimeout`, which is set to 5 (it maps to `--connect-timeout` flag), so this change shouldn't affect failing connection.

Reply:
Ah, my bad, all good then!